docs/pkp/wallets/sendingTxs.md

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_position: 3
+---
+
+# Sending Transactions with PKPs
+
+:::note
+**STATE OF THE NETWORK**
+
+PKPs are still heavily in development and should NOT BE USED to transact live assets that hold monetary value. DO NOT store assets you are not prepared to lose. During this period, we're grateful for feedback on how to improve the docs and examples.
+:::
+
+You can use the [Lit PKP SDK](https://github.com/LIT-Protocol/lit-pkp-sdk) (an extension of the PKPWallet package) to handle Ethereum JSON RPC requests for signing messages, data, and transactions and sending transactions with PKPs.
+
+The SDK supports the following signing methods:
+
+- eth_sign
+- personal_sign
+- eth_signTypedData
+- eth_signTypedData_v1
+- eth_signTypedData_v3
+- eth_signTypedData_v4
+- eth_signTransaction
+- eth_sendTransaction
+- eth_sendRawTransaction
+
+## Examples
+
+### Signing a Transaction
+
+```js
+
+// Transaction to sign
+const from = address;
+const to = address;
+const gasLimit = BigNumber.from('21000');
+const value = BigNumber.from('10');
+const data = '0x';
+// pkp-ethers signer will automatically add missing fields (nonce, chainId, gasPrice, gasLimit)
+const txParams = {
+  from,
+  to,
+  gasLimit,
+  value,
+  data,
+};
+
+// eth_signTransaction parameters
+// Transaction - Object
+// Reference: https://ethereum.github.io/execution-apis/api-documentation/#eth_signTransaction
+const payload = {
+  method: 'eth_signTransaction',
+  params: [txParams],
+};
+
+// Initialize Lit PKP Wallet
+const wallet = new LitPKP({
+  pkpPubKey: publicKey,
+  controllerAuthSig: authSig,
+  provider: 'https://rpc-mumbai.maticvigil.com',
+});
+await wallet.init();
+
+// Sign eth_signTransaction request
+const result = await wallet.signEthereumRequest(payload);
+console.log('eth_signTransaction result', result);
+
+```
+
+### Sending a Transaction
+
+``` js
+// Transaction to sign and send
+const from = address;
+const to = address;
+const gasLimit = BigNumber.from('21000');
+const value = BigNumber.from('10');
+const data = '0x';
+// pkp-ethers signer will automatically add missing fields (nonce, chainId, gasPrice, gasLimit)
+const txParams = {
+  from,
+  to,
+  gasLimit,
+  value,
+  data,
+};
+
+// eth_sendTransaction parameters
+// Transaction - Object
+// Reference: https://ethereum.github.io/execution-apis/api-documentation/#eth_sendTransaction
+const payload = {
+  method: 'eth_sendTransaction',
+  params: [txParams],
+};
+
+// Initialize Lit PKP Wallet
+const wallet = new LitPKP({
+  pkpPubKey: publicKey,
+  controllerAuthSig: authSig,
+  provider: 'https://rpc-mumbai.maticvigil.com',
+});
+await wallet.init();
+
+// Handle eth_sendTransaction request
+const result = await wallet.signEthereumRequest(payload);
+console.log('eth_sendTransaction result', result);
+
+```
+
+### eth_sign
```

Comment:
Any chance we can highlight `personal_sign` instead? `eth_sign` is an oldish spec, and folks are pushing for `personal_sign` and `eth_signTypedData_v4`. Learn more [here](https://docs.metamask.io/guide/signing-data.html#signtypeddata-v4).